ShoppingList/ShoppingList/ShoppingItem.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ShoppingList
+{
+    public class ShoppingItem
+    {
+
+        public ShoppingItem(string v1)
+        {
+            this.Name = v1;
```

Comment:
```suggestion
            Name = name;
```
the 'this' keyword is used to reference the current object to distinguish between an object's data members and data members being passed into that object. Since your property name does not match your param name you do not need to use the 'this' keyword as there is no ambiguity. 

I would also suggest renaming your string parameter to 'name' to make your code more readable as it isn't clear what v1 stands for.